ckanext/xloader/jobs.py

```diff
@@ -324,7 +363,7 @@ def callback_xloader_hook(result_url, api_key, job_dict):
     return result.status_code == requests.codes.ok
 
 
-def set_datastore_active_flag(data_dict, flag):
+def set_resource_metadata(data_dict, flag):
```

Comment:
I guess it's a bit inconsistent between how we pass datastore_active and datastore_contains_all_records_of_source_file now. How about we get rid of `flag` and just pass `update_dict` containing the two values?

Reply:
I agree, I changed it accordingly